woocommerce/wc-admin
# [WIP] Apple pay inbox notification

Fixes #5099

WIP, waiting on final CTA link from @pmcpinto 

Adds a notification that Apple Pay is now available on Stripe.

### Screenshots

![Screen Shot 2020-09-25 at 5 08 19 pm](https://user-images.githubusercontent.com/9312929/94250869-6086bc80-ff54-11ea-8074-cf8de466a792.png)

### Detailed test instructions:

- Set the site to a US location.
- Install and activate the [WooCommerce Stripe Payment Gateway](https://wordpress.org/plugins/woocommerce-gateway-stripe/)
- Go to the WooCommerce home screen.
- Ensure the `Apple Pay is now available on Stripe` is displayed in the inbox.

### Changelog Note:

Tweak: Inbox notification added for Apple Pay on Stripe.
